src/dotnet/Fable.Core/Import/Node/Stream.fs

```diff
@@ -0,0 +1,124 @@
+module rec Fable.Import.Node.Stream
+
+open Fable.Core
+open Fable.Import.JS
+open Fable.Import.Node.Buffer
+open Fable.Import.Node.Events
+
+module stream_types =
```

Comment:
What I mean  is, put `[<AutoOpen>]` on it, and then it will open if one opens parent module.